Assets/Mirror/Runtime/Transport/FallbackTransport.cs

```diff
@@ -0,0 +1,137 @@
+// uses the first available transport for server and client.
+// example: to use Apathy if on Windows/Mac/Linux and fall back to Telepathy
+//          otherwise.
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Mirror
+{
+    public class FallbackTransport : Transport
```

Comment:
Perhaps add a [HelpURL("...")]  Nobody will have a clue what this does unless we explain it somewhere.

Reply:
done

Reply:
Shouldn't this be Fallback.html?

Reply:
This should be Fallback.html, not Multiplexer.html